samples/ftd_configuration/access_rule_with_applications.yml

```diff
@@ -0,0 +1,36 @@
+- hosts: all
+  connection: httpapi
+  tasks:
+    - name: Find a Google Drive application
+      ftd_configuration:
+        operation: getApplicationList
+        filters:
+          name: Google Drive
+        register_as: google_app_results
+
+    - name: Find a Dropbox application
+      ftd_configuration:
+        operation: getApplicationList
+        filters:
+          name: Dropbox
+        register_as: dropbox_app_results
+
+    - name: Create an access rule allowing traffic from cloud file storages
+      ftd_configuration:
+        operation: upsertAccessRule
+        data:
+          name: Allow traffic from cloud file storages
+          ruleAction: PERMIT
+          eventLogAction: LOG_BOTH
+          embeddedAppFilter:
+            applications:
+              - id: "{{ dropbox_app_results[0].id }}"
```

Comment:
should we use - `dropbox_app_results[0].['id']` as we did in the previous playbook to follow the same style?

Reply:
Good point, we used both styles in different playbooks and it was not consistent. I checked and updated all playbooks to have dot notation as it's shorter.